server/webapp/WEB-INF/rails.new/app/assets/stylesheets/structure.scss

```diff
@@ -271,6 +271,10 @@ body[dir='ltr'] {
   padding: 10px;
 }
 
+#MB_content .modal-box-section {
```

Comment:
Avoid using id selectors<br>Selector `MB_content` should be written in lowercase with hyphens